plugins/zlux-agent/pluginDefinition.json

```diff
@@ -0,0 +1,35 @@
+{
+  "identifier": "org.zowe.zlux.agent",
+  "apiVersion": "1.0.0",
+  "pluginVersion": "1.0.0",
+  "pluginType": "application",
+  "dataServices": [
+    {
+      "type": "external",
+      "name": "fileapi",
+      "host": "agent",
+      "port": "8542",
+      "urlPrefix": "/",
+      "isHttps": true,
+      "version": "1.0.0"
+    },
+    {
+      "type": "external",
+      "name": "security-mgmt-api",
+      "host": "agent",
+      "port": "8542",
+      "urlPrefix": "/",
+      "isHttps": true,
+      "version": "1.0.0"
+    },
+    {
+      "type": "external",
+      "name": "server-plugins-api",
```

Comment:
the current server-plugins-api yaml document is 100% correct for zlux, and 50% correct for zss.
therefore, it's not correct to put it into `org.zowe.zlux.agent`.
it would be correct to make a copy of server-plugins-api yaml that only includes the `/server/agent` routes, and put that here.
then, zlux itself can serve the entire one, which includes `/server/agent`, because it proxies to agent.

Reply:
yes, separated into separate swagger for agent &server